databricks_asset_bundles/00.create_source_data.py

```
@@ -0,0 +1,100 @@
+from pyspark.sql import SparkSession
+from pyspark.sql.types import *
```

Ruff (0.7.0)

`from pyspark.sql.types import *` used; unable to detect undefined names

(F403)